troubleshooting/tools/tracevol.py

```diff
@@ -114,44 +140,53 @@ def format_table(arg, pvc_data, table):
         table.add_row([pvcname, "", "", False,
                        False, False])
         return
-    pool_name = get_pool_name(arg, volume_name)
+    pool_name = get_pool_name(arg, volume_name, isRBD)
     if pool_name == "":
         table.add_row([pvcname, pvname, "", False,
                        False, False])
         return
     # get image id
+    # for cephfs, image_id means subvolume id
```

Comment:
Better than a comment would be to rename the variable, but it can probably slide for now until a later refactoring.

Reply:
For RBD image_id makes more sense, that's why preserved the existing name. Same function serves for both cephfs and rbd.